packages/gatsby/src/utils/feedback.ts

```diff
@@ -0,0 +1,74 @@
+import report from "gatsby-cli/lib/reporter"
+import { getConfigStore, getGatsbyVersion } from "gatsby-core-utils"
+import latestVersion from "latest-version"
+
+const feedbackKey = `feedback.disabled`
+const lastDateKey = `feedback.lastRequestDate`
+
+// This function is designed to be used by `gatsby feedback --disable`
+// and `gatsby feedback --enable`. This key is used to determine
+// if a user is allowed to be solicited for feedback
+export function setFeedbackDisabledValue(enabled: boolean): void {
+  getConfigStore().set(feedbackKey, enabled)
+}
+
+// Print the feedback request to the user
+export function showFeedbackRequest(): void {
+  getConfigStore().set(lastDateKey, Date.now())
+  report.log(
+    `\n\nHello! Will you help Gatsby improve by taking a four question survey?`
```

Comment:
I might change this to be a bit more direct e.g. "Please help Gatsby improve by taking a four question survey! It'll take less than five minutes and your ideas and feedback will be very helpful."

cc @shannonbux

Reply:
Sounds great!

Reply:
Let's go with a combo: "Will you help Gatsby improve by taking a four question survey? It takes less than five minutes and your ideas and feedback will be very helpful."